js/components/controlBar.js

```diff
@@ -879,33 +876,38 @@ var ControlBar = createReactClass({
         alignment = CONSTANTS.TOOLTIP_ALIGNMENT.LEFT;
       }
       tooltipAlignments[collapsedControlBarItems[k].name] = alignment;
-      finalControlBarItems.push(controlItemTemplates[collapsedControlBarItems[k].name]);
+      var item = controlItemTemplates[collapsedControlBarItems[k].name];
+      if (this.props.equalSpacing) {
+        item = (
+          <div className="oo-flex-row" key={collapsedControlBarItems[k].name}>
+            {item}
+          </div>
+        );
+      }
+      finalControlBarItems.push(item);
     }
+
     return finalControlBarItems;
   },
 
   render: function() {
     var controlBarClass = ClassNames({
       'oo-control-bar': true,
-      'oo-control-bar-hidden': !this.props.controlBarVisible
+      'oo-control-bar-hidden': !this.props.controlBarVisible,
+      'oo-control-bar-video': !this.props.simpleControlBar
```

Comment:
Could we use a different name for this prop? Without looking at the code it's not clear what will happen when this is set

Reply:
Will do